src/basic_recipes/text.jl

```diff
@@ -415,13 +415,13 @@ function max_x_advance(glyph_infos::Vector{GlyphInfo})::Float32
 end
 
 function max_y_ascender(glyph_infos::Vector{GlyphInfo})::Float32
-    return maximum(glyph_infos) do ginfo
+    maximum(glyph_infos; init = 0.0f0) do ginfo
```

Comment:
Why not -Inf?

Reply:
Then you have to insert a check prior to returning (though I guess that's more accurate)

Reply:
Why? Inf as return value for an empty list seems as valid as 0, no?